code/game/antagonist/antagonist_factions.dm

```diff
@@ -141,14 +141,24 @@
 		return
 
 	usr = user
-	var/text = input("Type message","[name] communication")
+	var/message = input("Type message","[name] communication")
 
-	if(!text || !is_member(user))
+	if(!message || !is_member(user))
 		return
 
-	text = capitalize_cp1251(sanitize(text))
+	message = capitalize_cp1251(sanitize(message))
+	var/text = "<span class='revolution'>[name] member, [user]: \"[message]\"</span>"
 	for(var/datum/antagonist/A in members)
-		A.owner.current << "<span class='revolution'>[name] member, [user]: \"[text]\"</span>"
+		A.owner.current << text
+
+	//ghosts
+	for (var/mob/M in GLOB.dead_mob_list)	//does this include players who joined as observers as well?
```

Comment:
I don't know, does it?

Reply:
dead_mob_list lists all dead mobs, and the below code should exclude clientless ( non-player or disconnected ) and excelsior messages will get sent to dead people.